src/parser/jobs/pld/modules/RoyalAuthority.tsx

```diff
@@ -0,0 +1,193 @@
+		this.currentSwordOath.used++
+	}
+
+	private onHolySpirit() {
+		if (this.divineMightActive === false) { return }
+		this.divineMightUses++
+	}
+
+	private onHolyCircle() {
+		if (this.divineMightActive === false) { return }
+		this.divineMightUses++
+	}
+
+	private onApplySwordOath(event: Events['statusApply']): void {
+		if (event.data == null) { return }
+
+		// Track potential uses & any overcap due to reapplication while the status was still active
+		if (event.data === this.data.statuses.SWORD_OATH.stacksApplied) {
+			if (this.currentSwordOath != null) {
+				this.swordOathOvercap += this.currentSwordOath.stacks
+				this.onRemoveSwordOath()
+			}
+		}
+
+		if (this.currentSwordOath == null) {
+			this.currentSwordOath = {
+				initial: event.data,
+				stacks: event.data,
+				used: 0,
+			}
+		} else {
+			this.currentSwordOath.stacks = event.data
+		}
+	}
+
+	private onRemoveSwordOath(): void {
+		if (this.currentSwordOath == null) { return }
+		this.swordOathHistory.push(this.currentSwordOath)
+		this.currentSwordOath = undefined
+	}
+
+	private onApplyDivineMight(): void {
+		this.divineMightActive = true
+		this.divineMightTotal++
+	}
+
+	private onRemoveDivineMight(): void {
+		this.divineMightActive = false
+	}
+
+	private onComplete() {
+		this.onRemoveSwordOath()
+		this.onRemoveDivineMight()
+
+		const swordOathStacks = this.data.statuses.SWORD_OATH.stacksApplied
+		const swordOathStacksUsed = this.swordOathHistory.reduce((used, swordOath) => used + swordOath.used, 0)
+		const swordOathPotentialStacks = this.swordOathHistory.reduce((potential, swordOath) => potential + swordOath.initial, 0)
+		// Leaving the following commented out since it's not actually being used right now, but here's the calculation for checking dropped stacks if that data becomes useful
+		// const droppedStacks = this.swordOathHistory.reduce((dropped, swordOath) => dropped + Math.max(swordOath.initial - swordOath.used, 0), 0)
+
+		this.checklist.add(new TieredRule({
+			name: <Trans id= "pld.sword-oath.checklist.name">Use Sword Oath stacks generated by Royal Authority</Trans>,
+			description: <Trans id="pld.sword-oath.checklist.description">
+				<DataLink action="ROYAL_AUTHORITY" /> generates {swordOathStacks} stacks of <DataLink status="SWORD_OATH" /> to use on <DataLink action="ATONEMENT" />.
+				This is effectively the same as getting {swordOathStacks} uses of <DataLink showIcon={false} action="ROYAL_AUTHORITY" />.
+			</Trans>,
+			tiers: SWORD_OATH_SEVERITY,
+			requirements: [
+				new Requirement({
+					name: <Trans id="pld.sword-oath.checklist.requirement.name">
+						Uses of <DataLink status="SWORD_OATH" /> out of possible uses
+					</Trans>,
+					overrideDisplay: `${swordOathStacksUsed} / ${swordOathPotentialStacks} (${this.getPercent(swordOathStacksUsed, swordOathPotentialStacks).toFixed(2)}%)`,
+					percent: this.getPercent(swordOathStacksUsed, swordOathPotentialStacks),
+				}),
+			],
+		}))
+
+		this.checklist.add(new TieredRule({
```

Comment:
these two should probably be combined into a single metric - if keeping as a checklist, probably some form of "use all resources granted by royal authority" - unless someone's got a strong call for keeping seperate?

Reply:
This may be a personal preference thing, but I like the idea for keeping them separate because they are two separate resources. Missing a divine might stack has a bigger dps loss than missing sword oath, but there are also fewer divine might stacks so the missing one is a bigger hit to the severity whereas missing a sword oath is harder to trigger because there are more instances.

I ultimately think it works out and would struggle to want to group them because they represent two different issues. Maybe someone is good at using all their divine might stacks but not sword oath, or the other way around?